client-src/README.md

```diff
@@ -0,0 +1,275 @@
+# Logistics Wizard Demo
+
+[![Build Status](https://travis-ci.org/davezuko/react-redux-starter-kit.svg?branch=master)](https://travis-ci.org/davezuko/react-redux-starter-kit?branch=master)
```

Comment:
@colbycheeze 

these come from the starter kit, we should update them to our build or just remove them.